Makefile

```diff
@@ -1,8 +1,18 @@
 IMAGE_NAME=quay.io/theforeman/candlepin
-IMAGE_TAG=4.4.14
+IMAGE_TAG=4.4.20-1
+
+PROJECT_XY_TAG=4.4
+PROJECT_XYZ_TAG=4.4.20
+
+FOREMAN_XY_TAG=3.15
+FOREMAN_XYZ_TAG=3.15.0
 
 build:
-	podman build -f images/candlepin/Containerfile -t ${IMAGE_NAME}:${IMAGE_TAG} .
+	podman build --file images/candlepin/Containerfile --tag ${IMAGE_NAME}:${IMAGE_TAG} --build-arg TAG=${IMAGE_TAG}
```

Comment:
Why not
```
podman pull quay.io/candlepin/candlepin:${TAG}
podman tag quay.io/candlepin/candlepin:${IMAGE_TAG} ${IMAGE_NAME}:${IMAGE_TAG}
```

The `build` feels weird given we're not actually building.
Or is that a setup to allow changes in the future?